docs/api/web-contents.md

```diff
@@ -1079,6 +1079,15 @@ The factor must be greater than 0.0.
 
 Returns `Number` - the current zoom factor.
 
+#### `contents.setTextZoomFactor(factor)`
+
+* `factor` Double - Text Zoom factor; default is 1.0.
+
+Changes the text zoom factor to the specified factor. Text Zoom factor is only
+text zoom percent divided by 100, so 300% = 3.0.
+
+The factor must be greater than 0.0.
+
 **[Deprecated](modernization/property-updates.md)**
```

Comment:
This note belongs to the `contents.getZoomFactor()` method  above.

Reply:
ok i rebase to master and fix docs error